seesaw/script/run_pipeline.py

```diff
@@ -191,7 +200,7 @@ def main():
     parser.add_argument("--port", dest="port_number",
                         help="the port number for the web interface "
                              "(default: 8001)",
-                        metavar="PORT", type=int, default=8001)
+                        metavar="PORT", type=int, default=get_available_port())
```

Comment:
The help string should be updated to indicate the default is not just 8001 but a range of available ports (to follow the "Principle of Least Astonishment").